src/plugins/explore/public/components/visualizations/bar/to_expression.ts

```diff
@@ -164,6 +169,7 @@ export const createTimeBarChart = (
   // Configure bar mark
   const barMark: any = {
     type: 'bar',
+    opacity: 0.8,
```

Comment:
I think all `opacity: 0.8,` should be removed, I see this value in different place